modular_citadel/code/game/objects/items/melee/holy_weapons.dm

```diff
@@ -0,0 +1,17 @@
+obj/item/nullrod/monkeyidol
+	name = "Golden monkey idol"
+	desc = "Great authority dwells within this idol"
+	icon_state = "Monkeyidol"
+	item_state = "Monkeyidol"
+	icon = 'modular_citadel/icons/obj/holyweapons.dmi'
+	force = 5
+	throwforce = 20
```

Comment:
> 20 throwforce

Reply:
@Poojawa >never heard of the atheists fedora